nutrition_labels/finding_most_common_references.py

```diff
@@ -0,0 +1,39 @@
+import json
+import pandas as pd
+
+if __name__ == '__main__':
+
+    # Read in data
+
+    reference_data = []
+    for row in open('data/processed/tech_grants_references_dict_2020-09-16.json', "r"):
+       reference_data.append(json.loads(row))
+
```

Comment:
I get an error when I run this
`json.decoder.JSONDecodeError: Unterminated string starting at: line 1 column 351 (char 350)`
It fails on the row:
```
'{"id": "19847070", "source": "MED", "citationType": "JOURNAL ARTICLE", "title": "Far transfer in cognitive training of older adults.", "authorString": "Zelinski EM.", "journalAbbreviation": "Restor. Neurol. Neurosci.", "issue": "5", "pubYear": 2009, "volume": "27", "pageInfo": "455-471", "citedOrder": 59, "match": "Y", "essn": "1878-3627", "issn": "0922-'
```
which seems to abruptly end and didnt finish the dict, I think means that json hadn't finished being saved to maybe?

I wonder whether your local version of "tech_grants_references_dict_2020-09-16.json" is the same as the one on sharepoint?

Reply:
when i opened this json in a text editor it turns out this is the last entry. It seems to me that maybe when "tech_grants_references_dict_2020-09-16.json" was created it might have not fully finished saving?

Reply:
So I just checked the one on sharepoint and its about 30MB smaller than the one i have locally 
I will try uploading it again (there might be a file limit) but also I just tried to find that row in the tech_grats_references_dict that I have locally and I cant find it

Reply:
ok cool, maybe the file hadn't fully uploaded to sharepoint then. Odd its not found locally though :/ Either way, hopefully when it's uploaded it will solve the issue

Reply:
So I just re downloaded it and the full thing dowloaded I dont know whats going on